### PR TITLE
Delete akborana.is-a.dev

### PR DESCRIPTION
**Pull request by akborana3**
Delete akborana.is-a.dev via Discord

### CODE REVIEW

**Review by Stef-DP — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/akay.json
Comment (orxngc) [inline domains/akay.json:1]: This site is inaccessible, please provide a preview.

File: domains/akborana1.json
Comment (orxngc) [inline domains/akborana1.json:1]: This site is inaccessible, please provide a preview.

Comment (akborana3) [reply domains/akborana1.json:1]: akborana.tech here is preview

**Pull request closed without merging**